torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -34,14 +34,17 @@ def get_dimensions(image: Union[PIL.Image.Image, torch.Tensor, features.Image])
     return list(get_chw(image))
 
 
-def get_image_num_channels(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> int:
+def get_num_channels(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> int:
     num_channels, *_ = get_chw(image)
     return num_channels
 
 
-def get_image_size(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> List[int]:
-    _, *image_size = get_chw(image)
-    return image_size
+get_image_num_channels = get_num_channels
```

Comment:
Should we also properly deprecate this? If not, could you please add comment about the aliasing like we did for the other aliases?

https://github.com/pytorch/vision/blob/6b2e0a084536b7d7abced67839b52258d6e21712/torchvision/prototype/transforms/functional/_geometry.py#L93-L96